salt/journal/init.sls

```diff
@@ -6,48 +6,61 @@ maintenance-mode-start:
         - require:
             - nginx-server-service
 
-journal-php-extensions:
-    cmd.run:
-        - name: apt-get install -y php7.0-redis
-        - require:
-            - php
-        - watch_in:
-            - service: php-fpm
-
-journal-repository:
-    builder.git_latest:
-        - name: git@github.com:elifesciences/journal.git
-        - identity: {{ pillar.elife.projects_builder.key or '' }}
-        - rev: {{ salt['elife.rev']() }}
-        - branch: {{ salt['elife.branch']() }}
-        - target: /srv/journal/
-        - force_fetch: True
-        - force_checkout: True
-        - force_reset: True
-        - fetch_pull_requests: True
-        - require:
-            - cmd: composer
-            - maintenance-mode-start
-
+journal-folder:
     file.directory:
         - name: /srv/journal
         - user: {{ pillar.elife.deploy_user.username }}
         - group: {{ pillar.elife.deploy_user.username }}
         - recurse:
             - user
             - group
+
+journal-folder-old-git-repository:
+    file.absent:
+        - name: /srv/journal/.git
         - require:
-            - builder: journal-repository
+            - journal-folder
+
+journal-docker-compose-env:
+    file.managed:
+        - name: /srv/journal/.env
+        - source: salt://journal/config/srv-journal-.env
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - template: jinja
+        - require:
+            - journal-folder
+
+journal-docker-compose-containers-env:
+    file.managed:
+        - name: /srv/journal/containers.env
+        - source: salt://journal/config/srv-journal-containers.env
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - template: jinja
+        - require:
+            - journal-folder
 
 config-file:
     file.managed:
-        - name: /srv/journal/app/config/parameters.yml
+        - name: /srv/journal/parameters.yml
+        # TODO: rename
```

Comment:
The source file?

Reply:
`salt://journal/config/srv-journal-app-config-parameters.yml` as it doesn't reflect the host's path anymore